data/NV/incentives.json

```diff
@@ -135,37 +141,19 @@
     ],
     "item": "weatherization",
     "program": "nv_wellsRuralElectric_saveEnergy,SaveMoney",
-    "amount": {
-      "type": "dollars_per_unit",
-      "number": 12,
-      "unit": "square_foot"
-    },
-    "owner_status": [
-      "homeowner"
-    ],
-    "short_description": {
-      "en": "Site built home and manufactured homes may qualify for a fractional rebate for window replacements, up to 100% of costs. U-factor .23 to .30."
-    }
-  },
-  {
-    "id": "NV-8",
-    "authority_type": "utility",
-    "authority": "nv-wells-rural-electric",
-    "payment_methods": [
-      "rebate"
-    ],
-    "item": "weatherization",
-    "program": "nv_wellsRuralElectric_saveEnergy,SaveMoney",
     "amount": {
       "type": "dollars_per_unit",
       "number": 16,
-      "unit": "square_foot"
+      "unit": "square_foot",
+      "minimum": 12,
+      "maximum": 16
     },
     "owner_status": [
       "homeowner"
     ],
     "short_description": {
-      "en": "Site built home and manufactured homes may qualify for a fractional rebate for window replacements, up to 100% of costs. U-factor .22 or lower."
+      "en": "Site built home and manufactured homes may qualify for a rebate for window replacements, up to 100% of costs. Up to 16/sq ft depending on U-factor.",
+      "es": "Las casas construidas en el sitio y las casas prefabricadas pueden obtener un reembolso fraccionado para la sustitución de ventanas, de hasta el 100% de los costos. Factor U de .23 a .30."
```

Comment:
this translation was written before I combined the 2 incentives - I reached out to our translator to get it updated (same for the NV-9 Spanish description)